.lagoon.env

```diff
@@ -1,3 +1,6 @@
 # Shield credentials (note these are placeholders replaced during provisioning).
 DRUPAL_SHIELD_USER=
 DRUPAL_SHIELD_PASS=
+
+# Do not remove. This location may be used by deployment scripts for database and config snapshots.
+SNAPSHOTS=/app/web/sites/default/files/private/backups
```

Comment:
this variable is not used anywhere

also, there should be a default value provided in the docker-compose.yml in case if this is removed

Reply:
Does lagoon read variables from docker-compose? From memory no.

Reply:
We’re considering having this standard in the govcms8lagoon container as well

Reply:
We’re considering having this standard in the govcms8lagoon container as well

Reply:
In lagoon ecosystem, variables are not read from `docker-compose.yml`. So the defaults should be placed into the `Docker.cli`.

The overrides can be placed into `.env*` file.

Reply:
Created issue ready to go https://github.com/govCMS/govcms8lagoon/issues/114